### PR TITLE
Common tsdocs

### PR DESCRIPTION
**Pull request by SepehrGanji**

### CODE REVIEW

File: packages/common/src/utils/array.ts [hunk 1]
```diff
@@ -52,6 +65,19 @@ export function hasDuplicatesBy<T>(array: T[], selector: ObjectSelector<T>): boo
...
+ * Turns an array into chunks of the specified size
+ * @param array
+ * @param size
+ *
+ * @example
+ * ```
+ * const array = [1, 2, 3, 4, 5];
+ * const chunks = chunk(array, 2);
+ * console.log(chunks);
+ * // [[1, 2], [3, 4], [5]]
```
Comment (arobsn) [inline packages/common/src/utils/array.ts:78]: Sweet!

File: packages/common/src/utils/array.ts [hunk 2]
```diff
@@ -81,6 +121,22 @@ export function orderBy<T>(
   });
 }
 
+/**
+ * Returns true if arrays are equal
```
Comment (arobsn) [inline packages/common/src/utils/array.ts:125]: What do you think about using "Checks" instead of "Returns true" for this and other cases?

**Review by arobsn — CHANGES_REQUESTED** [review]
Great job!

- Left some comments in the code
- Please use [@throw](https://tsdoc.org/pages/tags/throws/) tag when applicable

File: packages/common/src/utils/array.ts (hunk 1 above)
Comment (SepehrGanji) [reply packages/common/src/utils/array.ts:78]: Thanks!

File: packages/common/src/utils/array.ts (hunk 2 above)
Comment (SepehrGanji) [reply packages/common/src/utils/array.ts:125]: Yeah that's a good idea,
Refactored!

**Review by arobsn — APPROVED** [review]

**Pull request merged**